Gems/ROS2/Code/Source/Manipulation/ManipulationUtils.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <ROS2/Manipulation/JointInfo.h>
+
+namespace ROS2::Utils
+{
+    struct JointStateData
+    {
+        float position{ 0.f };
+        float velocity{ 0.f };
+        float effort{ 0.f };
```

Comment:
In Inlude/ROS2/ManipulationJointInfo.h there are aliases declared for joint position, velocity and effort (they are still float). They can technically be used here, but in the same moment they seem to be not used in the whole project. I would either update this code here or remove those aliases.

Reply:
I was considering using them, but decided against it since as you've mentioned, they're not used elsewhere. It's true that their usage is inconsistent and should somehow be unified.